inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -654,7 +669,7 @@ void RemovePermutationsNHWCToNCHWPass::run() {
         auto next = getInputTo(l->outData.front()).begin()->second;
         auto prev = CNNNetPrevLayer(l);
 
-        if (!LayerInfo(next).isPermute() || !LayerInfo(prev).isPermute()) {
+        if (!isIdentityPermuteCanBeRemoved(next, false) || !isIdentityPermuteCanBeRemoved(prev, true)) {
```

Comment:
i meant identity permute just as an example discribing type of permutes we are searching - not sure what tipe of permutes you are removing now - previously it was NCWH to  NHWC permutes as i remember